packages/utils/src/fhir-converter/shared.ts

```diff
@@ -84,6 +84,12 @@ export function getPatientIdFromFileName(fileName: string) {
   return parts.patientId;
 }
 
+export function getCxIdFromFileName(fileName: string) {
```

Comment:
getCxIdFromFileNameOrCreate might be better -- the typing will make it look like you're going to throw if you can't find it (i.e. `getCxIdFromFileNameOrFail`)